source/frontend/src/utils/mapPropertyUtils.ts

```diff
@@ -144,3 +147,59 @@ function toMapProperty(
     name: feature?.properties?.NAME,
   };
 }
+
+export function featuresetToMapProperty(
+  featureSet: LocationFeatureDataset,
+  address: string = 'unknown',
+): IMapProperty {
+  const pimsFeature = featureSet.pimsFeature;
+  const parcelFeature = featureSet.parcelFeature;
+  const regionFeature = featureSet.regionFeature;
+  const districtFeature = featureSet.districtFeature;
+
+  const propertyId = pimsFeature?.properties.PROPERTY_ID;
+  const pid = pidFromFeatureSet(featureSet);
+  const pin = pinFromFeatureSet(featureSet);
+  return {
+    propertyId: propertyId ? Number.parseInt(propertyId?.toString()) : undefined,
```

Comment:
number.parseInt vs Number()?

Reply:
Looks like number() is less lenient on the parameter and can result in a floating, while parse int only return integers